spyder/plugins/shortcuts/widgets/summary.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, parent=None):
         # Calculate font and amount of elements in each column
         # according screen size
         width, height = self.get_screen_resolution()
-        font_size = height / 80
+        font_size = height // 80
```

Comment:
```suggestion
        font_size = int(height / 80)
```

I think this is the right fix. Otherwise we could end up using the wrong font size below.

Reply:
I'm not sure I understand what you mean.  `height // 80` is equivalent to `math.floor(height / 80)`, but `int(height / 80)` might do rounding or truncation, depending on the behaviour of the C library (see note (3) to the table in https://docs.python.org/3/library/stdtypes.html#numeric-types-int-float-complex).  (Note (1) of that table says that `x // y` is an integer value but not necessarily of type `int`; in the case that both operands are of type `int`, as in the situation here, the result is of type `int`.)  It seems to make sense here to be explicit as to what is wanted: if truncation/floor is wanted, then `height // 80` does the job, as would `math.floor(height / 80); if rounding is preferred, then `int(round(height / 80))` would be needed.

Reply:
>  if rounding is preferred, then int(round(height / 80)) would be needed

Ok, I changed my suggestion accordingly.

Reply:
Let's go with `int(round(height / 80))`.

Reply:
OK, just pushed a commit making this change.